src/components/sysdetect/cpu.c

```diff
@@ -119,8 +119,10 @@ fill_cpu_info( _sysdetect_cpu_info_t *info )
     }
 
     for (a = 0; a < info->threads * info->cores * info->sockets; ++a) {
-        CPU_CALL(cpu_get_attribute_at(CPU_ATTR__HWTHREAD_NUMA_AFFINITY, a, &info->numa_affinity[a]),
-                 info->numa_affinity[a] = 0);
```

Comment:
Instead of setting numa_affinity[a]=0 in case of error we could have set it to -1 here?